wikipedia/challenges/default.json

```
@@ -43,9 +43,30 @@
     {
       "name": "default-search-application-search",
       "operation": "default-search-application-search",
-      "clients": {{application_search_clients | default(20)}},
-      "time-period": {{search_time_period | default(300) | int}},
-      "warmup-time-period": {{search_warmup | default(10) | int}}
+      "clients": {{application_search_clients | default(20) | int}},
+      "time-period": {{application_search_time_period | default(300) | int}},
+      "warmup-time-period": {{application_search_warmup_time_period | default(10) | int}}
+    },
+    {
+      "parallel": {
+        "completed-by": "parallel-documents-indexing-query-string-search",
+        "tasks": [
+          {
+            "name": "parallel-documents-indexing-bulk",
+            "operation": "parallel-documents-indexing",
+            "clients": {{ parallel_indexing_bulk_clients | default(1) | int }},
+            "warmup-time-period": {{parallel_indexing_bulk_warmup_time_period | default(10) | int}},
+            "target-throughput": {{ parallel_indexing_bulk_target_throughput | default(1) | int }}
```

Nit: remove whitespace, for consistency.
```suggestion
            "clients": {{parallel_indexing_bulk_clients | default(1) | int}},
            "warmup-time-period": {{parallel_indexing_bulk_warmup_time_period | default(10) | int}},
            "target-throughput": {{parallel_indexing_bulk_target_throughput | default(1) | int}}
```